### ISSUE TITLE
When using DAdaptAdam, training appears to learn ~nothing

### ISSUE THREAD

**Issue by JenXIII** [derrian-distro/LoRA_Easy_Training_Scripts#95]
I've tried 4 different trainings with DAdaptAdam (toml files attached), and compared to a reference AdamW8bit training (_5) using the same dataset (slight fixes in the _10 training, but very minor) it appears to do nothing each time.  For _7, _8, and _9 the only difference is weight decay value.  Between _9 and _10 I ran update.bat so _10 ought to be on the latest in main branch.

Am I doing something wrong or is this just broken?

[toml files.zip](https://github.com/derrian-distro/LoRA_Easy_Training_Scripts/files/11932942/toml.files.zip)
![xyz_grid-1031-12345-(masterpiece), (best quality), (ultra-detailed), photorealistic, (best illustration), (best shadow_0 2), (an extremely delicate](https://github.com/derrian-distro/LoRA_Easy_Training_Scripts/assets/24497594/a4612a50-1de4-4d5a-bd72-998346c69117)
Tensorboard logging for _10
![tensorboard_kamikaze_outfit_10](https://github.com/derrian-distro/LoRA_Easy_Training_Scripts/assets/24497594/18bc9dc5-671b-45f5-815a-850cc6889521)
![chrome_2023-07-02_21-31-31](https://github.com/derrian-distro/LoRA_Easy_Training_Scripts/assets/24497594/9ab5af9d-e108-47e7-bf29-20dcd0781966)

**Comment by derrian-distro** [comment]
in my experience it either learns nothing at all, or burns to high hell. so it's probably just Dadapt being shitty. that being said, I'm pretty sure Dadapt wants *very* specific setups which I definitely don't know. seems like it might not even be adjusting the lr correctly. This is much more of a kohya issue than it is a me issue, as I just hand it off to kohya to train. so I suggest posting this over there as well.

**Comment by knzro** [comment]
I checked the toml files, it seems you missed to add the extra parameter decouple=true, also the recommended scheduler is constant. They again my results were utterly ass. I tried with LR=1 DR=1 

Someone else around suggested .5 learning rate and decay from .2~.8 but I didn't get to test it any further out of seer disappointment, as it took me about 25%~30% extra training time making the experiment not very worth it.
